spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb

```diff
@@ -175,6 +182,34 @@ def setup_ems_and_cassette(refresh_settings)
         end
       end
 
+      it "is consistent with old refresh" do
```

Comment:
lets do also other spec turning this around, first graph refresh, then old refresh

Reply:
Sure thing. Will do.

Reply:
Done